sqe-database-access/Queries/EditionQueries.cs

```diff
@@ -162,6 +162,141 @@ internal class Result
 		}
 	}
 
+	internal class EditionQuery
+	{
+		private const string _baseQuery = @"
+SELECT DISTINCTROW ed1.edition_id AS EditionId,
```

Comment:
I don't know all the MySQL extensions to SQL. DISTINCTROW is another one of those. However, according to the documentation, it is exactly the same as DISTINCT. So why not just use DISTINCT, which everybody understands?